src/strategies/index.ts

```diff
@@ -446,11 +446,8 @@ const strategies = {
   'lydia-gov-vault': lydiaGovVault,
   'xkawa-farm': xkawaFarm,
   'darkforest-score': darkforestScore,
-  'orange-reputation-based-voting': orangeReputationBasedVoting,
-  'squid-dao': squidDao,
   'botto-dao': bottoDao,
-  genart,
-  'path-balance-staked-and-locked': pathBalanceStakedAndLocked
+  'volt-voting-power':voltVotingPower
```

Comment:
Something is wrong with this PR it removes some of the strategies

Reply:
ok..will look into that